xAPI.md

```diff
@@ -544,9 +552,22 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 <a name="xapi-components" />
 
-## 5.0 xAPI Components
+<a name="xapi_ecosystem"/>
+## 5.0 The xAPI Ecosystem
+<a name=handling_statements"/>
+### 5.1 Handling Statements<a name="handling_statements"/>
+xAPI is based on the assumption that information about learning experiences will be scattered across a multitude of entities that create, store and analyze it.   These different entities form a sort of ecosystem has three major players or components:
+
+1. [Statement Providers](#def_statement_provider) which are responsible for gathering and formatting the raw data about learning experiences and providing that data to Learning Record Stores;
+2. [Learning Record Stores](#def-learning-record-store) (LRS) which are responsible for storing and providing the data about the learning experiences.  In the xAPI specification, statements of learning experience are considered 
+to be evidence about what the learner has done or achieved.  It is the responsibility of the LRS to ensure that these statements are stored and provided back to consumers with exactly the same data that was provided.
```

Comment:
Take care not to duplicate the definitions section. This section shouldn't define **what** the "players" are. It should list who they are and describe how they interact.